flake-module.nix

```diff
@@ -129,8 +129,8 @@ in
       nixos-flake.lib = rec {
         inherit specialArgsFor;
 
-        mkLinuxSystem = mod: inputs.nixpkgs.lib.nixosSystem {
-          system = "x86_64-linux";
+        mkLinuxSystem = system: mod: inputs.nixpkgs.lib.nixosSystem {
```

Comment:
This looks like a regression to me. The system is already specified in `hardware-configuration.nix`, so this shouldn't be necessary.

Reply:
Hmm, I didn't realize `system` can be set in `nixpkgs.system`. PR incoming ...

https://github.com/NixOS/nixpkgs/blob/5b3975cc66cefd3c199505c7a5780c5bbba95bce/flake.nix#L30-L33

Reply:
https://github.com/srid/nixos-flake/pull/33